Blackjack.py

```diff
@@ -0,0 +1,282 @@
+#This code is based on the Udemy course milestone project
+#This code will take the Class and Card classes to make a 52 card deck
+#Will ask the player to make a bet agains the dealer and then play blackjack
+#Player will be asek to hit or take another card
+#This is a bit unique as I forgot how to play Blackjack and so the program keeps playing
+#until all 52 cards are used up

```

Comment:
The multiline comment format is more suited for documentation.
```
"""One line summary

More info
Can go here 

And it can take up paragraphs if needced.
"""
```
The `#` is for inline comments.

Reply:
And a couple typo's, but not really important for this.